src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -1158,6 +1158,10 @@ map.help.config = [[
             corresponding to the movement. As an example: us = {'upsouth',
             'downnorth', 0, -1, 1}
 
+        &lt;yellow&gt;custom_name_search&lt;reset&gt; - When this is set, instead of running the default
+            functon name_search, a global user-defined function called
+            'custom_name_search' is used instead. This may be true or false.
```

Comment:
How about the `mudlet` namespace and `mudlet.custom_name_search`? This is not to encourage global functions.

@keneanung ?

Reply:
Any idea about workaround? I can call "map.custom_name_search" extending the map namespace. Does it look better?

Reply:
I thought about `map` but that would mean your script always has to follow after the mapper in loading, a bit of a hassle

Reply:
Maybe we can create a special behaviur in mudlet namespace. Such as "mudlet.map.custom_name_search" or "mudlet.specialfunction.custom_name_search" or "mudlet.external.custom_name_search" or ... ?

Reply:
We already use a couple of variables, see https://wiki.mudlet.org/w/Mapping_script - `mudlet.mapper_script`, `mudlet.custom_speedwalk` - so `mudlet.custom_name_search` seems OK